src/main/scala/com/soundcloud/periskop/client/ExceptionAggregate.scala

```diff
@@ -1,12 +1,15 @@
 package com.soundcloud.periskop.client
 
+import java.time.ZonedDateTime
+
 import scala.collection.immutable.Queue
 
 private[client] case class ExceptionAggregate
 (
   totalCount: Long = 0,
   severity: Severity = Severity.Error ,
-  latestExceptions: Queue[ExceptionWithContext] = Queue.empty
+  latestExceptions: Queue[ExceptionWithContext] = Queue.empty,
+  timestamp: ZonedDateTime = ZonedDateTime.now()
```

Comment:
shall we call it `firstSeenTimestamp`? since it groups multiple exceptions occurrences

Reply:
happy to rename this one, should we rename in other clients too @marctc ?

Reply:
@jcreixell this in the internal name. In the UI its gonna by _First Occurrence_. What do you think? It's gonna be confusing?

Reply:
for me it is a bit confusing, a single exception has a timestamp, but a group of exceptions has many timestamps, so which one are we referencing to?

Reply:
In this case it would actually correspond to a timestamp, since it's set to `now()` - but you're right in that it's not so clear what it's being used for. @marctc shall we just rename this? Maybe something like `seenAt`, or `sentAt`?

Reply:
+1 to `seenAt` or `createdAt`

Reply:
I think `createdAt` is best 👍

Reply:
@blewis11 lets name it everywhere then